src/hooks/use-chat-context.tsx

```diff
@@ -53,7 +53,6 @@ const parseWpCliOutput = ( stdout: string, defaultValue: string[] ): string[] =>
 		const data = JSON.parse( stdout );
 		return data?.map( ( item: { name: string } ) => item.name ) || [];
 	} catch ( error ) {
-		console.error( error, stdout );
```

Comment:
This was logging to the browser console, but the user doesn't have access to that.